src/SFA.DAS.EmployerIncentives.Domain/ApprenticeshipIncentives/ApprenticeshipIncentive.cs

```diff
@@ -644,5 +645,67 @@ public void ValidateLearningData(Guid pendingPaymentId, Learner learner, Collect
             ValidateHasNoDataLocks(pendingPaymentId, learner, collectionPeriod);
             ValidateDaysInLearning(pendingPaymentId, learner, collectionPeriod, incentivePaymentProfiles);
         }
+
+        public void RequestEmploymentChecks(bool? isInLearning)
+        {
+            if (EmploymentChecks.Any())
+            {
+                return;
+            }
+
+            if (!isInLearning.HasValue || !isInLearning.Value)
+            {
+                return;
+            }
+
+            AddEmploymentChecks();
+        }
+
+        private void AddEmploymentChecks()
+        {
+            if (StartDate.AddDays(42) > DateTime.Now)
+            {
+                return;
+            }
+
+            AddEmploymentBeforeSchemeCheck();
+            AddEmployedAtStartOfApprenticeshipCheck();
+
+            AddEvent(new EmploymentChecksCreated(Id));
+        }
+
+        private void AddEmployedAtStartOfApprenticeshipCheck()
+        {
+            var secondCheck = EmploymentCheck.New(Guid.NewGuid(), Id, EmploymentCheckType.EmployedAtStartOfApprenticeship, StartDate, StartDate.AddDays(42));
+            Model.EmploymentCheckModels.Add(secondCheck.GetModel());
+        }
+
+        private void AddEmploymentBeforeSchemeCheck()
+        {
+            var phaseStartDate = GetPhaseStartDate();
+            var firstCheck = EmploymentCheck.New(Guid.NewGuid(), Id, EmploymentCheckType.EmployedBeforeSchemeStarted, phaseStartDate.AddMonths(-6), phaseStartDate.AddDays(-1));
+            Model.EmploymentCheckModels.Add(firstCheck.GetModel());
+        }
+
+        public void RefreshLearnerMatch(Learner learner)
+        {
+            SetHasPossibleChangeOfCircumstances(learner.HasPossibleChangeOfCircumstances);
```

Comment:
Can the methods these call be made private?

Reply:
SetHasPossibleChangeOfCircumstances is used elsewhere.  I've made the others private.